models/Document.php

```diff
@@ -1491,4 +1495,11 @@ public function setVersionCount(?int $versionCount): ElementInterface
 
         return $this;
     }
+
+    protected function getListingCacheKey(array $args = []) {
+        $unpublished = $args[0] ?? false;
```

Comment:
maybe cast to bool here

        $children = $doc->getChildren(true);
        $children = $doc->getChildren('true');

will return 2 different cache keys

Reply:
done!